CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## UNRELEASED
 
+### Added
```

Comment:
Keep-a-changelog suggests this as another category. Might we worth adopting?

```suggestion
### Changed
```

Reply:
I haven't been using it, so up to you.

Reply:
We can always change our minds.